todo.css

```diff
@@ -0,0 +1,36 @@
+body {
+  background-color: #FAECA9
+}
+
+.topBar {
+  font-family: sans-serif;
```

Comment:
put this line on the `body` so that it gets propagated down to all the nodes, so that you don't have to put it for `.list` as well.